lib/cache-control.js

```diff
@@ -46,7 +46,13 @@ function cacheControl (type, time) {
 }
 
 function preventCache () {
-  this.header('Cache-Control', 'no-store, max-age=0, private')
+  this
+    .header('Cache-Control', 'no-store, max-age=0, private')
+    // compatibility support for HTTP/1.0
+    // see: https://owasp.org/www-community/OWASP_Application_Security_FAQ#how-do-i-ensure-that-sensitive-pages-are-not-cached-on-the-users-browser
+    .header('pragma', 'no-cache')
+    .header('expires', 0)
```

Comment:
```suggestion
    .header('Pragma', 'no-cache')
    .header('Expires', 0)
```

To be consistent with the naming of `Cache-Control`